<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsEditorQt</class>
 <widget class="QMainWindow" name="PrefsEditorQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SheepShaver Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::TabShape::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="volumesTab">
       <attribute name="title">
        <string>Volumes</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QListView" name="volumesList"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="addVolumeButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="createVolumeButton">
              <property name="text">
               <string>Create</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="removeVolumeButton">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Shared Folder:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sharedFolderEntry"/>
            </item>
            <item>
             <widget class="QPushButton" name="sharedFolderSelectButton">
              <property name="text">
               <string>Browseâ€¦</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Boot from:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="bootFromCombo">
              <item>
               <property name="text">
                <string>Any</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CD-ROM</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>Use CD-ROM driver</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="avTab">
       <attribute name="title">
        <string>Audio/Video</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Start as:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="screenModeComboBox">
          <item>
           <property name="text">
            <string>Window</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fullscreen</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Resolution:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="resolutionComboBox">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string>512 x 384</string>
          </property>
          <item>
           <property name="text">
            <string>512 x 384</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>640 x 480</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>800 x 600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1024 x 768</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1280 x 800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1440 x 900</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1600 x 1200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1920 x 1080</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Maximum</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Refresh rate:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="refreshComboBox">
          <item>
           <property name="text">
            <string>5 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7.5 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>30 Hz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>60 Hz</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="quickDrawCheckBox">
          <property name="text">
           <string>QuickDraw acceleration</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="audioCheckBox">
          <property name="text">
           <string>Enable sound output</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="audioMixerLabel">
          <property name="text">
           <string>Audio mixer:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="audioDeviceLabel">
          <property name="text">
           <string>Audio output device:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="audioDeviceComboBox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="audioMixerComboBox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="inputTab">
       <attribute name="title">
        <string>Input</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QCheckBox" name="keycodeCheckBox">
          <property name="text">
           <string>Use keycode file:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLineEdit" name="keycodeFileEntry"/>
          </item>
          <item>
           <widget class="QPushButton" name="keycodeFileSelectButton">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QSpinBox" name="cursorLinesSpinBox">
            <property name="suffix">
             <string> lines</string>
            </property>
            <property name="minimum">
             <number>-99</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="mouseWheelModeCursorRadioButton">
            <property name="text">
             <string>Moves cursor:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QRadioButton" name="mouseWheelModePageRadioButton">
            <property name="text">
             <string>Scrolls page</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Mouse wheel:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="swapOptCmdComboBox">
          <property name="text">
           <string>Swap Mac OS modifier keys</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="portsTab">
       <attribute name="title">
        <string>Ports</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="1">
         <widget class="QComboBox" name="serialAComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="serialBComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Modem port:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Printer port:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Ethernet interface:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="ethernetComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>slirp</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="miscTab">
       <attribute name="title">
        <string>Memory/Misc</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Mac OS RAM size:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="ramSizeComboBox">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string>16 MB</string>
          </property>
          <item>
           <property name="text">
            <string>16 MB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>32 MB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>64 MB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>128 MB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>256 MB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>512 MB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1024 MB</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>ROM file:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLineEdit" name="romFileEntry"/>
          </item>
          <item>
           <widget class="QPushButton" name="romFileSelectButton">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="ignoreIllegalCheckBox">
          <property name="text">
           <string>Ignore illegal memory accesses</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QCheckBox" name="idleWaitCheckBox">
          <property name="text">
           <string>Don't use CPU when idle</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QCheckBox" name="noGuiCheckBox">
          <property name="text">
           <string>Show settings window on startup</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="jitTab">
       <attribute name="title">
        <string>JIT</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QCheckBox" name="native68kCheckBox">
          <property name="text">
           <string>Enable native 68k DR emulator (experimental)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="jitCheckBox">
          <property name="text">
           <string>Enable JIT compiler</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>&amp;Start</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>&amp;Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionSave"/>
    <addaction name="actionZap"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionWebsite"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionWebsite">
   <property name="text">
    <string>&amp;Website</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>&amp;Start SheepShaver</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save &amp;Preferences</string>
   </property>
  </action>
  <action name="actionZap">
   <property name="text">
    <string>&amp;Zap NVRAM File</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
